app/assets/stylesheets/base/variables.scss

```diff
@@ -6,13 +6,10 @@ it should be added here as a variable. Changing the value
 of these variables will affect the colors sitewide. */
 $futura-regular: "futura-pt", Arial, sans-serif;
 $futura-condensed: "futura-pt-condensed", Arial, sans-serif;
-$proxima-nova: "proxima-nova", Arial, sans-serif;
 $arial: Arial, sans-serif;
 
-$font-stack-body: $proxima-nova;
-$font-stack-body-new: $arial;
+$font-stack-body: $arial;
 $font-stack-heading: $futura-regular;
```

Comment:
Can you create a variable called `$font-stack-title` to replace the `$font-stack-heading-new` that was removed? Or maybe switch the rules this way if it makes more sense, up to you:

```
$font-stack-title: $futura-regular;
$font-stack-heading: $futura-condensed;
```
Then use those variables throughout all the other stylesheets. Not `$futura-condensed` or `$futura-regular`. The variables with the font names in them are only used in this file to create the more generic variables, which then get used in all the other stylesheets. This is so that when we change fonts in the future, it's just this one file where it needs to be updated, and everywhere that has the variables `$font-stack-body`, `$font-stack-heading`, etc. still make sense.

Reply:
Makes total sense to me – good catch.

I have a suggestion here with the naming. Since they both apply to headings, perhaps we could word it like this so it's clear that they apply to headings and are similar in nature:

```
$font-stack-heading: $futura-regular;
$font-stack-heading-emphasis: $futura-condensed;
```

The word "emphasis" could be whatever you want – bold, large, impact, etc. Let me know if you like this approach. If not, I'll stick to the original request.

Reply:
@sharissis-brei Yes, that's a a great idea! Any one of those you suggested, like emphasis or bold, are perfect. Whatever you think fits.

Reply:
fyi, emphasis, in the html world, usually leans more towards meaning italics. Terms like strong or important are associated boldness and font weight. Referencing this article https://www.w3schools.com/html/html_formatting.asp. I don't think we need to necessarily change it if you've already made the change, unless you or Kayla think it would be clearer.

Reply:
@mandythomas That's true, good point. I was thinking about it from more of a conceptual angle, but I can see how it might be confusing. I'm happy to change it to whatever works best for you guys, just let me know.

Reply:
I talked to Ross about where the condensed font is used, trying to see if we could have a variable name in the scss that better described it's purpose than the -emphasis in $font-stack-heading-emphasis. He said it's mostly for the homepage for those short 2-word headings at the top of the Hero Panel and the Undergraduate Facts panel eg. the Experiment Boldly phrase. I'm drawing a blank on the right word to describe that so unless Kayla objects or has a better suggestion I guess just leave it at $font-stack-heading-emphasis.